src/lib/PreviewUI.js

```diff
@@ -112,6 +115,10 @@ class PreviewUI {
         this.container = this.container.querySelector(SELECTOR_BOX_PREVIEW_CONTAINER);
         this.previewContainer = this.container.querySelector(SELECTOR_BOX_PREVIEW);
         this.contentContainer = this.container.querySelector(SELECTOR_BOX_PREVIEW_CONTENT);
+        this.customLogoEl = this.container.querySelector(SELECTOR_BOX_PREVIEW_LOGO_CUSTOM);
```

Comment:
We should try to avoid adding instance properties when possible. It seems like we should be able to handle all of this within `setupHeader`.